Examples/TextEditorApp/TextEditorApp/TextEditor/TextViewModel.swift

```diff
@@ -75,10 +78,7 @@ class TextViewModel {
                 events.filter { $0.actor != clientID }.forEach {
                     switch $0.type {
                     case .content:
-                        let range = NSRange(location: $0.from, length: $0.to - $0.from)
-                        let content = $0.content ?? ""
-
-                        textChanges.append(.edit(range: range, content: content))
+                        break
```

Comment:
Logic has been modified. Just double checking if this was done on purpose.

Reply:
That logic was replaced by syncText in line 105. Thanks.